Geometry/VeryForwardData/data/CTPPS_Diamond_2021/CTPPS_Diamond_Detector_Assembly.xml

```diff
@@ -9,6 +9,7 @@
   <SolidSection label="CTPPS_Diamond_Detector_Assembly.xml">
     <!-- Detector Main Box and Planes -->
     <Box name="CTPPS_Diamond_Main_Box" dx="[CTPPS_Diamond_Parameters:PCB_dx]/2-[Sensors_Displacement_Foil]" dy="[CTPPS_Diamond_Parameters:Plane_dy]/2" dz="[box_dz]/2"/>
+    <Box name="CTPPS_Diamond_Main_Short" dx="[RP_Box:RP_Box_Height]/2-[Sensors_Displacement_Foil]" dy="[CTPPS_Diamond_Parameters:Plane_dy]/2" dz="[box_dz]/2"/> 
```

Comment:
Is this ok also for the Reco version in which we have bioth cylindrical pots? This file is loaded also for Reco one.

Reply:
In principle they are OK with the overlap given in geometryRPFromDD2021, so the diamonds are there with the new box (not the short one). I'll check with Valentina just to be sure, although she would check the files.